test/terminfo.jl

```diff
@@ -887,21 +907,21 @@ let
         :xvpa => false)
 
 @testset "terminfo" begin
-    dumb = Base.TermInfo(read(IOBuffer(dumb_terminfo), Base.TermInfoRaw))
+    dumb = Terminfo.TermInfo(read(IOBuffer(dumb_terminfo), Terminfo.TermInfoRaw))
```

Comment:
```
Error in testset terminfo:
2024-02-12 23:03:17 UTC	Error During Test at C:\buildkite-agent\builds\win2k22-amdci6-4\julialang\julia-master\julia-67bbd97d72\share\julia\test\terminfo.jl:909
2024-02-12 23:03:17 UTC	  Got exception outside of a @test
2024-02-12 23:03:17 UTC	  UndefVarError: `Terminfo` not defined in `Main.Test54Main_terminfo`
2024-02-12 23:03:17 UTC	  Stacktrace:
2024-02-12 23:03:17 UTC	    [1] macro expansion
2024-02-12 23:03:17 UTC	      @ C:\buildkite-agent\builds\win2k22-amdci6-4\julialang\julia-master\julia-67bbd97d72\share\julia\test\terminfo.jl:910 [inlined]
```
```suggestion
    dumb = Base.TermInfo(read(IOBuffer(dumb_terminfo), BaseTermInfoRaw))
```

Reply:
Ooops, this slipped in by accident while experimenting with "make terminfo handling a package".